src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1117,6 +1117,7 @@ protected override void GetItem(string path)
 
             try
             {
+#if !UNIX
                 bool retrieveStreams = false;
                 FileSystemProviderGetItemDynamicParameters dynamicParameters = null;
```

Comment:
If that makes the code cleaner overall, then I'm fine to have the code compiled on Unix.